TWLight/users/views.py

```diff
@@ -693,6 +694,14 @@ def get(self, request, pk, version, format=None):
         # We're ignoring streams here, because the API operates at the partner
         # level. This is fine for the use case we built it for (Wikilink tool)
         valid_partner_auths = get_valid_partner_authorizations(pk)
+
+        # List only those users who have been active lately,
+        # that is, do not list the users who had last logged in more than two weeks ago
+        if partner.authorization_method == partner.BUNDLE:
+            valid_partner_auths = valid_partner_auths.filter(
+                user__editor__user__last_login__gt=timezone.now() - timedelta(weeks=2)
+            )
+
```

Comment:
This code as it stands will fire `get_valid_partner_authorizations(pk)` whether it will be used or not.
Here's a suggested fix that you can commit directly into the PR.

Let's put `get_valid_partner_authorizations` an `else` and update the comments to better reflect the fact that we're dealing with a list of auths, not users.
```suggestion
        # We're ignoring streams here, because the API operates at the partner
        # level. This is fine for the use case we built it for (Wikilink tool)
        valid_partner_auths = None

        # For Bundle partners, get auths for users who logged in within the last 2 weeks.
        if partner.authorization_method == partner.BUNDLE:
            valid_partner_auths = valid_partner_auths.filter(
                user__editor__user__last_login__gt=timezone.now() - timedelta(weeks=2)
            )
        # Otherwise, get all valid auths for this partner.
        else:
            valid_partner_auths = get_valid_partner_authorizations(pk)
```

Reply:
also, I just noticed that you were using `user__editor__user__` in your query filter.  I streamlined that to `user__` to eliminate unnecessary db queries.